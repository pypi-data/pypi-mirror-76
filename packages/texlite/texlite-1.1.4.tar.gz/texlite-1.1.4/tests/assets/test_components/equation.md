In-line maths: $e^{\pi i} + 1 = 0$

Here's a single-line equation:

$$$
E = mc^2
$$$

Here are multiple lines:

$$$
x = 4
y = 2x + 1
\therefore y = 9
$$$

Here's are multiple lines aligned:

$$$
2x - 5 &= 8
3x + 9y &= -12
$$$