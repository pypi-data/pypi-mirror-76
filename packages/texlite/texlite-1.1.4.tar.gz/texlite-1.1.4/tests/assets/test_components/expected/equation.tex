% meta
\documentclass[10pt]{extarticle}

% packages
\usepackage[utf8]{inputenc}
\usepackage[margin=1.6in]{geometry}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}

% preamble commands
\linespread{1.0}
\usepackage{graphicx}
\graphicspath{{tests/assets/test_components/}}

% document details
\date{}

% ------------------------------------------------------------------------------
\begin{document}
% ------------------------------------------------------------------------------

In-line maths: $e^{\pi i} + 1 = 0$

Here's a single-line equation:

\begin{align*}
E = mc^2 \\
\end{align*}

Here are multiple lines:

\begin{align*}
x = 4 \\
y = 2x + 1 \\
\therefore y = 9 \\
\end{align*}

Here's are multiple lines aligned:

\begin{align*}
2x - 5 &= 8 \\
3x + 9y &= -12 \\
\end{align*}

% ------------------------------------------------------------------------------
\end{document}
% ------------------------------------------------------------------------------

